jordan_bundy/test/greet_test.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+var expect = require('chai').expect;
+var greet = require(__dirname + '/../lib/greet');
+// Create a dummy process to test the command line capability
+var process = {argv: ['node', 'greet.js', 'some name']};
```

Comment:
You can pass `process.argv[2]` into your testing function so you don't need to make a dummy process (altho it works, so...)